pkg/kubelet/kubelet_network_linux.go

```diff
@@ -37,13 +37,14 @@ const (
 )
 
 func (kl *Kubelet) initNetworkUtil() {
-	iptClients := []utiliptables.Interface{
-		utiliptables.New(utiliptables.ProtocolIPv4),
-		utiliptables.New(utiliptables.ProtocolIPv6),
+	iptClients := utiliptables.NewDualStack()
+	if len(iptClients) == 0 {
+		klog.InfoS("No iptables support on this system; not creating the KUBE-IPTABLES-HINT chain")
```

Comment:
for ipv4 it also installs the firewall rule for KUBE-FIREWALL

Reply:
That's a legacy rule that is not guaranteed to stay around forever anyway. (Kube-proxy iptables creates its own separate copy.) KEP-3178 [suggested we might eventually drop that rule](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3178-iptables-cleanup#indeterminate-future) and the blog post also hinted that ["`KUBE-FIREWALL` ... may eventually go away"](https://kubernetes.io/blog/2022/09/07/iptables-chains-not-api/#upcoming-changes). So I don't think we need to mention it.

Reply:
Ok